components/DatasetDetails/DatasetAboutInfo.vue

```diff
@@ -2,37 +2,177 @@
+                :class="{
+                  'active-citation': activeCitation === 'chicago-note-bibliography'
+                }"
+                @click="handleCitationChanged('chicago-note-bibliography')"
+              >
+                Chicago
+              </a>
+              |
+              <a
+                title="Format citation ieee"
+                :class="{ 'active-citation': activeCitation === 'ieee' }"
+                @click="handleCitationChanged('ieee')"
+              >
+                IEEE
+              </a>
+              |
+              <a
+                :href="`https://citation.crosscite.org/?doi=${doiValue}`"
+                target="_blank"
+              >
+                More on Crosscite.org
+              </a>
+            </div>
+          </el-col>
+        </el-row>
       </div>
       <h3>Tags</h3>
-      <ul>
-        <li>
-          Tag 1
-        </li>
-        <li>
-          Tag 2
-        </li>
-      </ul>
+      <div v-if="datasetTags.length !== 0">
+        <tag-list :tags="datasetTags" />
+      </div>
+      <div v-else>
+        <p>N/A</p>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
+import TagList from '@/components/TagList/TagList.vue'
+import { propOr } from 'ramda'
 export default {
   name: 'DatasetAboutInfo',
+
+  components: {
+    TagList
+  },
   props: {
-    datasetDetails: {
-      type: Object,
-      default: () => {}
+    updatedDate: {
+      type: String,
+      default: ''
+    },
+
+    doi: {
+      type: String,
+      default: ''
+    },
+
+    doiValue: {
+      type: String,
+      default: ''
+    },
+
+    datasetRecords: {
+      type: Array,
+      default: () => []
+    },
+
+    datasetTags: {
+      type: Array,
+      default: () => []
+    }
+  },
+
+  data() {
+    return {
+      citationLoading: false,
+      citationText: '',
+      activeCitation: '',
+      crosscite_host: process.env.crosscite_api_host
+    }
+  },
+
+  computed: {
+    /**
+     * Return DOI link
+     * @returns {String}
+     */
+    DOIlink: function() {
+      return this.doiValue ? `https://doi.org/${this.doiValue}` : ''
+    }
+  },
+
+  watch: {
+    DOIlink: {
+      handler: function(val) {
+        if (val) {
+          this.handleCitationChanged('apa')
+        }
+      },
+      immediate: true
+    }
+  },
+
+  methods: {
+    /**
+     * gets bibiolography based on citation type for current DOI
+     * @param {String} citationType
+     */
+    handleCitationChanged: function(citationType) {
+      if (citationType === this.activeCitation) {
+        return
+      }
+      this.citationLoading = true
+      this.activeCitation = citationType
+      // find all citation types at https://github.com/citation-style-language/style
+      const url = `${this.crosscite_host}/format?doi=${this.doiValue}&style=${citationType}&lang=en-US`
+      return fetch(url)
+        .then(response => {
+          return response.text()
+        })
```

Comment:
this is a total nitpick, but I find these one liners easier to read without the return statement `.then(response => response.text())`

no need to fix...just a stylistic comment

Reply:
Thanks matt! I'll keep in mind :)